source/index-management.rst

```diff
@@ -669,6 +669,13 @@ Common API Components
      * filter expression are included in the index.
      */
     partialFilterExpression: Document;
+
+    /**
+     * Optionally specifies a collation to use for the index. If not specified, the default collation
```

Comment:
I think I would expect the following from the Java API: 

* If a user doesn't provide any options when creating an index, I would create the options object with the collections collation settings.
* If a user provides options, then there wouldn't be any default for the collation settings, just what the user has set.